src/sentry/static/sentry/app/views/substances/substances.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {substancesGet} from 'app/redux/actions/substance';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css';
+import LoadingError from 'app/components/loadingError';
+import LoadingIndicator from 'app/components/loadingIndicator';
+
+class Substances extends React.Component {
+  componentWillMount() {
+    this.props.getSubstances();
+  }
+
+  render() {
+    if (this.props.loading) {
+      return <LoadingIndicator />;
+    } else if (this.props.errorMessage) {
+      return <LoadingError />;
+    }
+
+    return (
+      <div>
+        <ReactTable
+          data={this.props.substances}
+          columns={[
+            {
```

Comment:
These columns will come from the API and will not be in the UI like this. Just want to get a preview out to trunk as soon as possible.